googleapis/google-cloud-node
# migrate code from googleapis/nodejs-vmmigration

- feat: initial stub of library
- feat: initial generation of library (#1)
- chore: release 0.2.0 (#7)
- docs(node): support "stable"/"preview" release level (#1312) (#13)
- chore: add api_shortname and library_type to repo metadata (#12)
- chore(deps): update dependency @types/node to v16 (#8)
- chore(deps): update dependency sinon to v12 (#11)
- chore(deps): update dependency mocha to v9 (#10)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#15)
- test(nodejs): remove 15 add 16 (#1322) (#17)
- build: update copyright year to 2022 (#19)
- chore: update v2.12.0 gapic-generator-typescript (#21)
- chore(deps): update dependency sinon to v13 (#23)
- docs(nodejs): version support policy edits (#1346) (#25)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#26)
- chore: update v2.14.2 gapic-generator-typescript (#30)
- feat: promote to stable (#31)
- chore(main): release 1.0.0 (#32)
- build(node): update client library version in samples metadata (#1356) (#39)
- chore(deps): update dependency sinon to v14 (#42)
- build!: update library to use Node 12 (#44)
- chore(main): release 2.0.0 (#45)
- chore(deps): update dependency jsdoc-region-tag to v2 (#48)
- chore(deps): update dependency jsdoc-fresh to v2 (#47)
- feat: support regapic LRO (#49)
- chore(deps): update dependency linkinator to v4 (#52)
- chore(main): release 2.1.0 (#50)
- feat: add upgrade appliance method, update other methods (#53)
- fix: better support for fallback mode (#55)
- fix: change import long to require (#56)
- chore(main): release 2.2.0 (#54)
- fix: use _gaxModule when accessing gax for bundling (#59)
- chore(deps): update dependency uuid to v9 (#61)
- fix: do not import the whole google-gax from proto JS (#1553) (#58)
- fix: preserve default values in x-goog-request-params header (#62)
- test: use fully qualified request type name in tests (#64)
- fix(deps): use google-gax v3.5.2 (#68)
- chore(deps): update dependency @types/node to v18 (#66)
- chore(deps): update dependency jsdoc to v4 (#70)
- fix: regenerated protos JS and TS definitions (#71)
- chore(main): release 2.2.1 (#60)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
